code/game/machinery/cryopod.dm

```diff
@@ -455,6 +455,9 @@
 	//Eject any items that aren't meant to be in the pod.
 	var/list/items = src.contents
 	if(occupant)
+		if(usr != occupant && !occupant.client && occupant.stat<2)
```

Comment:
Related to previous conversation
>Occupant is living and has no client.

actually means `occupant.stat != DEAD` ~~(or may be `! (occupant.stat & DEAD)`)~~
Anyway 2 is magick number.
```suggestion
		if(usr != occupant && !occupant.client && occupant.stat != DEAD)
```

Reply:
Well, I tried to reuse code, as calling `incapacitated()` would be abused with just using `sleep` verb
https://github.com/discordia-space/CEV-Eris/blob/907d44d858d5fe0644724a7b8da7936f73e9eac1/code/__defines/mobs.dm#L154-L166
As there is no value for DEAD case. Honestly, have no idea how would it check if mob is dead and took the easiest option was available

Reply:
Check "Suggested change".

Reply:
This is what I used on the excelsior chat to share the contents to ghosts with AntagHUD enabled.